src/tile/tile.scss

```diff
@@ -211,10 +211,24 @@
       position: absolute;
       bottom: $iui-baseline * 0.5;
       right: $iui-s;
-      visibility: hidden;
+
+      clip-path: inset(50%);
+      overflow: hidden;
+      white-space: nowrap;
+      height: 1px;
+      width: 1px;
+      padding: 0;
+      margin: -1px;
+      border-width: 0;
```

Comment:
Why not a reusable mixin for this?

Reply:
Created `@mixin iui-visually-hidden`.  Not sure if we can provide a reusable mixin for reverting because we don't know what values they should have for border, margin, padding, etc.

Reply:
The solution to that is apply it on a wrapper div instead.

See https://github.com/iTwin/synchronization-report-react/blob/main/src/components/utils/VisuallyHidden.scss

Reply:
Any source to recommendation for doing this?

Reply:
- https://webaim.org/techniques/css/invisiblecontent/#:~:text=receive%20keyboard%20focus.-,css%20clip,-%7Bclip%3A%20rect
- https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html
- https://www.a11yproject.com/posts/how-to-hide-content/

Reply:
If I did a wrapper div, does something like this make sense within `global.scss`:
```scss
.iui-visually-hidden {
  @mixin iui-visually-hidden;

  &:focus-within {
		@mixin iui-visually-hidden-revert;
	}
}
```

Reply:
Might be able to avoid a wrapper div with something like this:
```scss
.iui-tile-more-options {
  // ... base styles

  &:not(:focus):not(.iui-visible) {
    @include iui-visually hidden;
  }
}
```

Reply:
Ugh…I read this _after_ adding the wrapper. 😓 Check what I have now and tell me if I should revert it.

Worth noting, I can not figure out why the `State More Options hover` scenario test is failing.